package.json

```diff
@@ -32,6 +32,7 @@
     "fbjs": "^0.8.8",
     "hyphenate-style-name": "^1.0.2",
     "inline-style-prefixer": "^3.0.2",
+    "lodash": "^4.17.4",
```

Comment:
Please remove the lodash dependency. It adds too much to the bundle size even when using specific modules